### PR TITLE
*: apply some lint

### PR DESCRIPTION
**Pull request by xhebox**
Signed-off-by: xhe <xw897002528@gmail.com>

### What problem does this PR solve?

Issue Number: close None

Problem Summary: As title. One notable thing: struct rearrange may improve GC performance.

What is changed and how it works:

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Notable changes

- [ ] Has configuration change
- [ ] Has HTTP API interfaces change (Don't forget to [add the declarative for API](https://github.com/tikv/pd/blob/master/docs/development.md#updating-api-documentation))
- [ ] Has weirctl change
- [ ] Other user behavior changes

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,6 +1,6 @@
 module github.com/pingcap/TiProxy
 
-go 1.19
+go 1.18
```
Comment (djshow832) [inline go.mod:3]: Why?

**Review by djshow832 — COMMENTED** [review]
How do you know that struct rearrangement can improve GC performance? Is there any tool to rearrange it?

File: go.mod (hunk 1 above)
Comment (xhebox) [reply go.mod:3]: My another laptop is `1.18`. I modified it to pass my local test. Never mind, it will be modified back.

**Comment by xhebox** [comment]
> How do you know that struct rearrangement can improve GC performance? Is there any tool to rearrange it?

Gopls. More accurately, it is just possible to improve but not necessary. Check https://github.com/1pkg/gopium/issues/24#issuecomment-951203325.

File: go.mod (hunk 1 above)
Comment (xhebox) [reply go.mod]: Done.

**Review by djshow832 — APPROVED** [review]

**Pull request merged**